### PR TITLE
Removed the modules using deprecated v4 sdks

### PR DESCRIPTION
**Pull request by abhinavbansal29**
endpoints which are deprecated needs to be removed:
1. List all the iSCSI attachments associated with the given Volume Group
2. List all the VM attachments for a Volume Group
3. List all the category details that are associated with the Volume Group

### CODE REVIEW

File: tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml [hunk 1]
```diff
@@ -156,22 +156,24 @@
...
-    that:
-      - result.error == None
-      - result.response is defined
-      - result.changed == false
-      - result.response | length == 1
-      - result.response[0].ext_id == iscsi_client_uuid
-      - result.volume_group_ext_id == vg1_uuid
-    fail_msg: "Unable to list iscsi clients attached to VG"
-    success_msg: "ISCSI clients attached to VG listed successfully"
+# Need to verify attachment
```
Comment (bhati-pradeep) [inline tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml:159]: Remove it

Comment (bhati-pradeep) [reply tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml:159]: As its deprecated

Comment (bhati-pradeep) [reply tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml:159]: Also use VG info to check if attached client is present or not in VG info

File: tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml [hunk 2]
```diff
@@ -244,22 +248,24 @@
...
+
+# - name: List all clients attached to VG to verify attachment
```
Comment (bhati-pradeep) [inline tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml:253]: Same as above.

File: tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml [hunk 3]
```diff
@@ -280,21 +286,23 @@
...
-  ansible.builtin.assert:
-    that:
-      - result.error == None
-      - result.response is defined
-      - result.changed == false
-      - result.response | length == 0
-      - result.volume_group_ext_id == vg1_uuid
-    fail_msg: "Unable to list iscsi clients attached to VG"
-    success_msg: "ISCSI clients attached to VG listed successfully"
+# Need to verify detachment
```
Comment (bhati-pradeep) [inline tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml:289]: Same as above

File: tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml [hunk 4]
```diff
@@ -209,21 +211,23 @@
     fail_msg: "Unable to detach iscsi client using initiator name"
     success_msg: "ISCSI client detached successfully using initiator name"
 
-- name: List all clients attached to VG to verify detachment
-  ntnx_volume_groups_iscsi_clients_info_v2:
-    volume_group_ext_id: "{{ vg1_uuid }}"
-  register: result
+# Need to verify detachment
```
Comment (bhati-pradeep) [inline tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml:214]: Remove it. Use VG info to check if the attached client is not present in the VG info.

File: tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml [hunk 5]
```diff
@@ -259,21 +259,23 @@
     fail_msg: "Unable to detach VM from VG"
     success_msg: "VM detached from VG successfully"
 
-- name: Fetch all VMs attached to VG
-  ntnx_volume_groups_vms_info_v2:
-    ext_id: "{{ vg1_uuid }}"
-  register: result
+# Need to verify here that VM is detached from VG
```
Comment (bhati-pradeep) [inline tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml:262]: Use VG info to check if VM is attached or dettached. This is critical verification dont remove it

File: tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml [hunk 6]
```diff
@@ -293,20 +295,22 @@
...
 
-- name: Verify VM2 is detached from VG
-  ansible.builtin.assert:
-    that:
-      - result.error == None
-      - result.changed == false
-      - result.response | length == 0
+# - name: Fetch all VMs attached to VG
```
Comment (bhati-pradeep) [inline tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml:300]: same as above

File: tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml [hunk 7]
```diff
@@ -189,55 +189,55 @@
...
-  ansible.builtin.assert:
-    that:
-      - result.error == None
-      - result.changed == false
-      - result.response | length == 1
-      - result.response[0].ext_id == "{{ vm1_uuid }}"
-
-    fail_msg: "Unable to verify fetching VM attached using filter"
-    success_msg: "Fetching VM attached using filter verified successfully"
+# - name: Fetch all VMs attached to VG
```
Comment (bhati-pradeep) [inline tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml:192]: remove info tests

**Review by bhati-pradeep — COMMENTED** [review]
1. Update module doc if module is not removed but updated.
2. Remove module names from runtime.yml

File: tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml (hunk 4 above)
Comment (bhati-pradeep) [reply tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml:214]: VG info doesnt have attachments info. Ignore

File: tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml (hunk 2 above)
Comment (bhati-pradeep) [reply tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml:253]: VG info doesnt have attachments info. Ignore. Remove testcase

File: tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml (hunk 3 above)
Comment (bhati-pradeep) [reply tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml:289]: VG info doesnt have attachments info. Ignore. Remove testcase

File: tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml (hunk 5 above)
Comment (bhati-pradeep) [reply tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml:262]: VG info doesnt have attachments info. Ignore. Remove testcase

File: tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml (hunk 6 above)
Comment (bhati-pradeep) [reply tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml:300]: VG info doesnt have attachments info. Ignore. Remove testcase

File: tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml (hunk 3 above)
Comment (abhinavbansal29) [reply tests/integration/targets/ntnx_volume_groups_iscsi_clients_v2/tasks/iscsi_client_connections.yml]: Removed

File: tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml (hunk 5 above)
Comment (abhinavbansal29) [reply tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml]: Removed

File: tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml (hunk 6 above)
Comment (abhinavbansal29) [reply tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml]: Removed

File: tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml (hunk 7 above)
Comment (abhinavbansal29) [reply tests/integration/targets/ntnx_volume_groups_vms_v2/tasks/ahv_vm_vg_connections.yml]: Removed

**Review by bhati-pradeep — APPROVED** [review]

**Pull request merged**